### PR TITLE
Ammo updates to 1.6

### PR DESCRIPTION
**Pull request by N7Huntsman**
7.92x57mm Mauser
8x50mmR Lebel
9x39mm Soviet
.30 Carbine
44-40 Winchester
5.8x42mm DBP10
6.5x50mmSR Arisaka
7.5x54mm French
7.7x58mm Arisaka
.300 AAC Blackout
4.73x33mm Caseless
7.92x33mm Kurz
.30-06 Springfield
45-70 Govt.

### CODE REVIEW

File: Defs/Ammo/Rifle/77x58mmArisaka.xml [hunk 1]
```diff
@@ -45,39 +43,11 @@
 			<graphicClass>Graphic_StackCount</graphicClass>
 		</graphicData>
 		<statBases>
-			<MarketValue>0.11</MarketValue>
+			<MarketValue>0.12</MarketValue>
 		</statBases>
 		<ammoClass>FullMetalJacket</ammoClass>
 		<cookOffProjectile>Bullet_77x58mmArisaka_FMJ</cookOffProjectile>
 	</ThingDef>
-
```
Comment (NoImageAvailable) [inline Defs/Ammo/Rifle/77x58mmArisaka.xml]: What's with the deletion?

File: Defs/Ammo/Advanced/12x64mmCharged.xml [hunk 2]
```diff
@@ -33,6 +33,7 @@
     <tradeTags>
       <li>CE_AutoEnableTrade_Sellable</li>
     </tradeTags>
+    <stackLimit>5000</stackLimit>
```
Comment (LockdownX7) [inline Defs/Ammo/Advanced/12x64mmCharged.xml]: Duplicate field.

File: Defs/Ammo/Medieval/CrossbowBolts.xml [hunk 3]
```diff
@@ -1,244 +1,254 @@
...
+      <li>CE_AutoEnableCrafting_FueledSmithy</li>
+      <li>CE_AutoEnableCrafting_ElectricSmithy</li>
+    </tradeTags>
+  </ThingDef>
+
+  <ThingDef Class="CombatExtended.AmmoDef" ParentName="AmmoCrossbowBoltBase">
+    <defName>Ammo_CrossbowBolt_Stone</defName>
+    <label>crossbow bolt</label>
+    <graphicData>
+      <texPath>Things/Ammo/Neolithic/Bolt/Stone</texPath>
```
Comment (LockdownX7) [inline Defs/Ammo/Medieval/CrossbowBolts.xml]: Missing texture at this path.

File: Defs/Ammo/Neolithic/SlingBullet.xml [hunk 4]
```diff
@@ -1,213 +1,173 @@
...
+	<!-- The amount of arrows usable over the ammo lifetime for each resource cost is equal to
+		[r / (1 - x)];
+			x = preExplosionSpawnChance,
+			r = resource cost. -->
+
+	<ThingDef Class="CombatExtended.AmmoDef" ParentName="BaseSlingBullet">
+		<defName>Projectile_SlingBullet_Stone</defName>
+		<label>sling bullet (stone)</label>
+		<graphicData>
+			<texPath>Projectiles/Sling/Stone</texPath>
```
Comment (LockdownX7) [inline Defs/Ammo/Neolithic/SlingBullet.xml]: Missing texture at this path.

File: Defs/Ammo/Neolithic/SlingBullet.xml [hunk 5]
```diff
@@ -1,213 +1,173 @@
...
+			<preExplosionSpawnChance>0.166</preExplosionSpawnChance>	<!-- 11.99 stones per resource -->
+			<preExplosionSpawnThingDef>Ammo_SlingBullet_Stone</preExplosionSpawnThingDef>
+		</projectile>
+	</ThingDef>
+
+	<ThingDef Class="CombatExtended.AmmoDef" ParentName="BaseSlingBullet">
+		<defName>Projectile_SlingBullet_Steel</defName>
+		<label>sling bullet (steel)</label>
+		<graphicData>
+			<texPath>Projectiles/Sling/Steel</texPath>
```
Comment (LockdownX7) [inline Defs/Ammo/Neolithic/SlingBullet.xml]: Missing texture at this path.

Comment (N7Huntsman) [reply Defs/Ammo/Neolithic/SlingBullet.xml]: Uploaded new projectile texture, update path.

File: Defs/Ammo/Neolithic/SlingBullet.xml (hunk 4 above)
Comment (N7Huntsman) [reply Defs/Ammo/Neolithic/SlingBullet.xml]: Uploaded new projectile texture, update path.

**Pull request merged**